utils/documentOps.py

```diff
@@ -208,40 +190,39 @@ def print_md():
     print("\n# Supported ONNX Operation for Target *" + target_arch + "*.\n")
     # Top paragraph.
     print("Onnx-mlir currently supports ONNX operations targeting up to " +
-        "opset " + str(hightest_opset) + ". Limitations are listed when applicable.\n")
+        "opset " + str(hightest_opset) + ". Limitations are listed when applicable."+
+        " This documentation highlights the minimum and maximum opset versions that" +
+        " are fully supported by onnx-mlir and not the version changes." + "\n")
     print("* Operations are defined by the [ONNX Standard]" +
         "(https://github.com/onnx/onnx/blob/main/docs/Operators.md).")
-    print("* Opset indicates, for each operation, the ONNX opset that " +
-        "(1) last modified that operation and " +
-        "(2) is supported by the current version of onnx-mlir. " +
-        "For example, \"Add\" was modified in Opset 14 and carries on unmodified " +
-        "to Opset 16. If onnx-mlir supports Opset 14, we thus list \"14\" as the Opset " +
-        "associated with the \"Add\" operation.")
+    print("* **Supported Opsets** indicates the lowest and highest opset a model" +
+          " may have for onnx-mlir to support compiling a model with the operator. ")
+    print("   * A * indicates onnx-mlir is compatible with the latest" +
+          " version of that operator available as of opset " + str(hightest_opset) + ".")
+
     print("\n")
     # Additional top paragraph.
     if additional_top_paragraph:
         print(additional_top_paragraph)
         print("\n")
     # Table.
-    header = ["Op", "Min Opset", "Max Opset", "Limitations"]
-    separator = ["---", "---", "---", "---"]
+    header = ["Op", "Supported Opsets", "Limitations"]
```

Comment:
Could add "inclusive".. though its probably understood

Reply:
Better to be explicit. I'll add it.